mypyc/rt_subtype.py

```diff
@@ -0,0 +1,47 @@
+"""*Runtime* Subtype check for RTypes."""
```

Comment:
Discuss how this is different from normal subtype checks for RTypes and why do we need both.